packages/kit/src/views/Home/components/WalletBanner/WalletBanner.tsx

```
@@ -31,6 +31,7 @@ import {
 } from '@onekeyhq/shared/src/utils/openUrlUtils';
 import type { IWalletBanner } from '@onekeyhq/shared/types/walletBanner';
 
+import { useThemeVariant } from '../../../../hooks/useThemeVariant';
```

ESLint

[error] Too many parent references (4 while maximum of 3 allowed)

(import-path/parent-depth)